packages/python-packages/apiview-stub-generator/apiview_reqs.txt

```diff
@@ -1,2 +1,3 @@
 ./
-pylint-guidelines-checker==0.0.7
\ No newline at end of file
+pylint-guidelines-checker==0.0.7
+pylint<3.0.0
```

Comment:
curious on why this addition is needed?

Reply:
API review parser does not work with latest linter > 3.0 and it was released last month. Scott B has already set this package requirement is python repo when running api review parser. We didn't run into this issue so far since there was no change in parser to test this pipeline which is only triggered when modifying parser itself. We need to ensure pylint less than 3.0 is used to successfully run API parser in test pipeline.